doc/source/deploy-guide/overview.rst

```diff
@@ -62,13 +62,25 @@ Your environment should be setup with the following:
 
 * `deployer` must run openSUSE Leap 15 or SLE15. Those must have all the
   deployment tools available. See more details on the page
-  :ref:`targethosts`. If you are not coming with your own node for
-  `deployer`, this tooling can create one for you (currently only on OpenStack).
+  :ref:`targethosts`.
 
-* The :term:`CaaSP` cluster must run :term:`CaaSP` version 3. If you are not
-  coming with your own :term:`CaaSP` cluster, this tooling can create one for
-  you (currently only on OpenStack).
+  .. note::
+     If you are not coming with your own node for
+     `deployer`, this tooling can create one for you in an OpenStack
+     environment. However, this should be treated as experimental.
 
-* The :term:`SES` cluster must run :term:`SES` 5. If you are not coming with
-  your own :term:`SES` cluster, this tooling can create an "All-in-one" node
-  for :term:`SES` for you (currently only on OpenStack).
+* The :term:`CaaSP` cluster must run :term:`CaaSP` version 3.
+  :term:`CaaSP` must be updated to its latest 3 version.
```

Comment:
I would write that it is `CaaSP` version 3 and it requires all maintenance updates installed on all its nodes